src/games/pictionary/Pictionary.js

```diff
@@ -0,0 +1,109 @@
+import firebase from 'firebase';
+import React from "react";
+import GameComponent from "../../GameComponent.js";
+import Pictionary from "./PictionaryApp";
```

Comment:
Nit: Can you import `PictionaryApp` here instead to match the filename?

Reply:
@jtessler Yes this part has been annoying me here. Will figure out a better way and try to keep in line with what you have been doing in other games. PR coming shortly.